city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.isfauthority.com"]
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        'name': 'Authority offices',
+        'address': 'Guaranteed Rate Field, 333 West 35th Street, Chicago, IL'
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css('div.wpb_text_column div.wpb_wrapper div.inner-text h2'):
+            meeting = Meeting(
+                title=self._parse_h2_title(item),
+                description='',
+                classification=BOARD,
+                start=self._parse_h2_start(item),
+                end=None,
+                all_day=False,
+                time_notes='',
+                location=self._parse_location(item),
```

Comment:
A `_validate_location` function might make more sense here, more info in the [docs](https://cityscrapers.org/docs/development/#location)